The reason being that you're not using relative paths in the makefile: 

```
mkdir -p /usr/share/icons/ubuntu-mono-dark/status/22/
mkdir: cannot create directory '/usr/share/icons': Permission denied
Makefile:31: recipe for target 'install_icons' failed
make[1]: *** [install_icons] Error 1
```

Build log here: https://launchpadlibrarian.net/207921335/buildlog_ubuntu-utopic-amd64.indicator-lockkeys_0.0.0-0~utopic_BUILD:ING.txt.gz

With my changes, the package should be built properly on launchpad. Also, you can remove the "install" file from the debian folder (I forgot to do that in my commit) since the makefile now takes care of everything.

--- a/debian/install
+++ b/debian/install
@@ -1,5 +1,0 @@
-icons/light/* /usr/share/icons/ubuntu-mono-light/status/22/
-icons/dark/* /usr/share/icons/ubuntu-mono-dark/status/22/
-icons/dark/* /usr/share/icons/hicolor/22x22/status/
-indicator-lockkeys.desktop /etc/xdg/autostart/
-indicator-lockkeys /usr/bin/

